internal/pkg/aws/cloudformation/stackset/stackset.go

```diff
@@ -112,27 +112,66 @@ func (ss *StackSet) UpdateAndWait(name, template string, opts ...CreateOrUpdateO
 	return ss.WaitForOperation(name, id)
 }
 
-// DeleteAllInstances removes all stack instances from a stack set and returns the operation ID.
-// If the stack set does not exist, then return [ErrStackSetNotFound].
-// If the stack set does not have any instances, then return [ErrStackSetInstancesNotFound].
-// Both errors should satisfy [IsEmptyStackSetErr], otherwise it's an unexpected error.
-func (ss *StackSet) DeleteAllInstances(name string) (string, error) {
+func (ss *StackSet) getStackSetUniqueAccountsAndRegions(name string) ([]InstanceSummary, error) {
```

Comment:
nit: what do you think of just ⬇️ 
```suggestion
func (ss *StackSet) getInstanceSummary(name string) ([]InstanceSummary, error) {
```